modules/rate/rate.model.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
+const rateSchema = new mongoose.Schema({
+  rate: {
+    type: Number,
+    default: 0,
+  },
+  lastUpdatedDate: {
+    type: Date,
+    default: Date.now,
+  },
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  product: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Product',
+  },
+  show: Boolean,
```

Comment:
Why do we need this property?

Reply:
To be able to hide unfavorable ratings of course :)